html/Axit/js/tabs/tab-switching.js

```diff
@@ -0,0 +1,40 @@
+import {components} from './tabs-components.js';
+import {
+  highlightButton,
+  removeButtonActiveClass,
+  hideContent,
+  showElement,
+  findContent,
+  changeSectionHeight,
+} from './helper-functions.js';
+
+export function addTabSwitchingEffect() {
+  components.then(
+      ({tabsButtonsList, contentList, activeTab, tabs, buttons}) => {
+        (function initFirstTab() {
+        // eslint-disable-next-line no-param-reassign
+          activeTab.style.opacity = 1;
```

Comment:
no real reason for this, as user will not even see this animation as tabs not in first screen